code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -172,6 +172,11 @@
 	description = "Water blessed by some deity."
 	color = "#E0E8EF" // rgb: 224, 232, 239
 
+/datum/reagent/water/holywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(is_servant_of_ratvar(M))
+		M << "<span class='userdanger'>A darkness begins to spread its unholy tendrils through your mind, purging the Justiciar's influence!</span>"
```

Comment:
This is the message for making them normal neutral crew members again, right? So why is it 'UNHOLY DARKNESS'?